docker-entrypoint.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
 set -e
 
+if [ "$SELFSIGNED_CERTS" = 'true' ]; then
+    wget https://raw.githubusercontent.com/Telecominfraproject/wlan-cloud-ucentral-deploy/main/docker-compose/certs/restapi-ca.pem -O /usr/local/share/ca-certificates/restapi-ca-selfsigned.pem
```

Comment:
Do we aim to support the Docker compose deployment in an offline environment? If so this wouldn't work.

Reply:
I guess network connectivity will be a requirement anyway as soon as an AP should be connected from outside of the local network. But yes, this can still lead to problems in environments with restricted network access.
The only alternative I see here is to include the CA cert in the images directly and then if the variable is set to true copy it into the ca-certificates location and run update-ca-certificates in docker-entrypoint.sh. Wanted to avoid that first since in my opinion it shouldn't be part of the application, but this would at least remove the need for internet access when deploying. What do you think?